CCIP mainnet (#1905)
smartcontractkit/documentation: src/content/ccip/release-notes.mdx

import { Aside } from "@components"

## Chainlink CCIP is GA - 2024-04-24

Chainlink CCIP is now Generally Available (GA) on mainnet and testnet.

To support your development and implementation needs, we encourage you to reach out to our team of experts for guidance and support. For expert advice, visit the [Chainlink CCIP Contact form](https://chain.link/ccip-contact).

> Additionally, we are pleased to provide the [Chainlink CCIP local
> simulator](https://github.com/smartcontractkit/chainlink-local) to enhance your development workflow. This tool optimizes the development experience by allowing you to simulate Chainlink CCIP functionality locally within your Hardhat and Foundry projects. The simulator is designed to ensure that your contracts tested locally can transition smoothly to test networks without any modifications.

Reviewer: ```suggestion
Additionally, the [Chainlink CCIP local
simulator](https://github.com/smartcontractkit/chainlink-local) is available to enhance your development workflow with CCIP. This tool allows you to simulate Chainlink CCIP functionality locally within your Hardhat and Foundry projects. The simulator is designed so you can test your contracts locally and transition smoothly to test networks without any modifications.
```
